fastestimator/backend/flip.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import List, TypeVar, Union
+
+import numpy as np
+import tensorflow as tf
+import torch
+
+Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor, np.ndarray)
+
+
+def flip(tensor: Tensor, axis: int = 0) -> Tensor:
+    """Reverse the order of a given `tensor` elements along a given axis.
+
+    This method can be used with Numpy data:
+    ```python
+    n = np.array([[[0, 1], [2, 3]], [[4, 5], [6, 7]], [[8, 9], [10, 11]]])
+    b = fe.backend.flip(n,axis = [0])  # [[[8,  9], [10, 11]], [[4,  5], [6,  7]], [[0,  1], [2,  3]]]
```

Comment:
the type hint on axis is 'int' but all the examples show as a list of int. Which is it supposed to be?